src/platform/datapath_raw_socket.c

```diff
@@ -442,6 +442,7 @@ CxPlatResolveRoute(
     // We queue an operation on the route worker for NS because it involves network IO and
     // we don't want our connection worker queue blocked.
     //
+    Route->State = RouteResolving;
```

Comment:
Now you can remove the "Route->State = RouteResolving;" in the below code block right?

Reply:
Oh, I missed that.